src/DesktopTests/UIAutomation/TreeWalkerForTestTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
For consistency with the production code, can we move this into the DesktopTests/UIAutomation/TreeWalkers folder and the Axe.Windows.DesktopTests.UIAutomation.TreeWalkers namespace?